src/components/DataView/DataView.tsx

```diff
@@ -16,6 +16,10 @@ const StyledWrapper = styled.div`
 	display: flex;
 	flex-direction: column;
 
+	& > .noResults {
+		margin-left: 20px;

```

Comment:
Let's keep consistency between both sides. Please change this to have 20px both on the left and right.